src/createServer.js

```diff
@@ -1,3 +1,56 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { sendBadRequest } = require('./sendBadRequest');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const urlParams = new URL(req.url, `http://${req.headers.host}`);
+
+    const textToConvert = urlParams.pathname.substring(1);
+    const toCase = urlParams.searchParams.get('toCase');
```

Comment:
You could also use the following approach:
```suggestion
        const [textToConvert, queryToCase] = req.url.slice(1).split('?');
    const searchParams = new URLSearchParams(queryToCase);
    const toCase= searchParams.get('toCase');
```